tests/dataconverter/test_helpers.py

```diff
@@ -17,6 +17,9 @@
 #
 """Test cases for the helper functions used by the DataConverter."""
 
+# pylint: disable=deprecated-module
+# distutils will become deprecated with py3.12
+
 import xml.etree.ElementTree as ET
 import os
 from distutils import file_util
```

Comment:
We should start using:
```suggestion
from setuptools import distutils
```

and then using `distutils.file_util` in the code

Reply:
On the distutils, agree, implemented and tested your suggestion, tests run fine with this and no longer deprecation warnings